src/core/writers/schemas.ts

```diff
@@ -85,18 +74,23 @@ export const writeSchemas = async ({
   specsName: Record<string, string>;
   header: string;
 }) => {
-  await ensureFile(join(schemaPath, '/index.ts'));
+  const schemaFilePath = join(schemaPath, '/index.ts');
+  await ensureFile(schemaFilePath);
 
-  return Promise.all(
-    schemas.map((schema) =>
-      writeSchema({
-        path: schemaPath,
-        schema,
-        target,
-        isRootKey,
-        specsName,
-        header,
-      }),
-    ),
+  schemas.map((schema) =>
```

Comment:
should stay a promise all the be sure to wait that everything has been written

Reply:
Added!